.config/tsaoptions.json

```diff
@@ -0,0 +1,10 @@
+{
+    "instanceUrl": "https://devdiv.visualstudio.com/",
+    "template": "TFSDEVDIV",
+    "projectName": "DEVDIV",
+    "areaPath": "DevDiv\\NET Fundamentals\\Infrastructure\\Runtime\\SDL",
```

Comment:
I know this is a placeholder value until we figure out the right area path.  

@GrabYourPitchforks can you know the right value?  If not, do you know who would be the right person?